quirks/password-rules.json

```diff
@@ -222,6 +222,9 @@
     "metlife.com": {
         "password-rules": "minlength: 6; maxlength: 20;"
     },
+    "microsoft.com": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; allowed: [-@_#!&$`%*+()./,;~:{}|?>=<^'[]];"
```

Comment:
How did you determine this specific set of special characters? Was it something documented, or did you experiment?

Reply:
An error stating the password rules showed up while creating a new account.

Reply:
Here's a copy of the form pertaining to live.com that references the rules, they seem to be inline with what's been specified in the above rule.

![signuplive-password-rules](https://user-images.githubusercontent.com/5795115/88996129-79f3eb00-d2ba-11ea-8f46-422de51983c7.gif)